api/organisations/invites/views.py

```diff
@@ -58,6 +58,15 @@ def join_organisation_from_link(request, hash):
         raise PermissionDenied("Invite links are disabled.")
 
     invite = get_object_or_404(InviteLink, hash=hash)
+    organisation = Organisation.objects.get(id=invite.organisation.id)
+    subscription_metadata = organisation.subscription.get_subscription_metadata()
```

Comment:
Why are you doing another explicit lookup in the database here? You can just do: 

```python
invite.organisation.subscription.get_subscription_metadata()
```

if you need the organisation explicitly, just do `organisation = invite.organisation`. This will do a lookup in the database if it needs to (i.e. if a join wasn't done on the initial query).